### PR TITLE
Portfolio: Mobile Version

### PR DESCRIPTION
**Pull request by julie-ify**
## Portfolio: Work and About Section

In this project I: 
- Added work section to my portfolio
- Added about section to my portfolio
- Added style to sections
- Used CSS grid and flexbox to align my work properly
- Added section images to my portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+              </h2>
+              <ul class="work-tools">
+                <li class="work-tool">Ruby on rails</li>
+                <li class="work-tool">css</li>
+                <li class="work-tool">javascript</li>
+                <li class="work-tool">html</li>
+              </ul>
+              <button class="see-project-btn" type="button">See Project</button>
+            </div>
+          </div>
```
Comment (arslanbisharat) [inline index.html:74]: - [ ] The list of projects uses a ul tag to group them, and li per project

File: index.html [hunk 2]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+                Gain+Glory
+              </h2>
+              <ul class="work-tools">
+                <li class="work-tool">Ruby on rails</li>
+                <li class="work-tool">css</li>
+                <li class="work-tool">javascript</li>
+                <li class="work-tool">html</li>
+              </ul>
+              <button class="see-project-btn" type="button">See Project</button>
+            </div>
```
Comment (arslanbisharat) [inline index.html:89]: - [ ] The list of projects uses a ul tag to group them, and li per project

File: index.html [hunk 3]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h2>
+              <ul class="work-tools">
+                <li class="work-tool">Ruby on rails</li>
+                <li class="work-tool">css</li>
+                <li class="work-tool">javascript</li>
+                <li class="work-tool">html</li>
+              </ul>
+              <button class="see-project-btn" type="button">See Project</button>
```
Comment (arslanbisharat) [inline index.html:104]: - [ ] The list of projects uses a ul tag to group them, and li per project

File: index.html [hunk 4]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+            </div>
+          </div>
```
Comment (arslanbisharat) [inline index.html:122]: - [ ] The list of projects uses a ul tag to group them, and li per project

File: index.html [hunk 5]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+                Gain+Glory
+              </h2>
+              <ul class="work-tools">
+                <li class="work-tool">Ruby on rails</li>
+                <li class="work-tool">css</li>
+                <li class="work-tool">javascript</li>
+                <li class="work-tool">html</li>
+              </ul>
+              <button class="see-project-btn" type="button">See Project</button>
+            </div>
```
Comment (arslanbisharat) [inline index.html:137]: - [ ] The list of projects uses a ul tag to group them, and li per project

File: index.html [hunk 6]
```diff
@@ -48,6 +48,169 @@ <h1 class="headline-text">
...
+          </div>
+        </div>
```
Comment (arslanbisharat) [inline index.html:155]: - [ ] The list of projects uses a ul tag to group them, and li per project

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: styles.css [hunk 7]
```diff
@@ -116,3 +121,204 @@ a {
...
+  margin-right: 12px;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+  font-style: normal;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 12px;
+  color: #3a4a42;
+  color: #36b37e;
```
Comment (arslanbisharat) [inline styles.css:322]: - [ ] You have used color property twice

File: index.html (hunk 1 above)
Comment (julie-ify) [reply index.html:74]: Please do you mean the works?

**Review by AshakaE — APPROVED** [review]
### Hi @julie-ify , 

Your project is complete! 👏 ... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional suggestions
_Please tick checkboxes after making changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by julie-ify** [comment]
Ok thanks

**Pull request merged**